sample/App.fs

```diff
@@ -49,10 +61,30 @@ let view model dispatch =
 // |> Program.run
 
 [<LitElement("sample-app")>]
-let App() =
-    Hook.useHmr(hmr)
-    let _ = LitElement.init(fun config ->
-        config.useShadowDom <- false
-    )
-    let model, dispatch = Hook.useElmish(init, update)
-    view model dispatch
\ No newline at end of file
+let App () =
+    Hook.useHmr (hmr)
+    let _ = LitElement.init (fun config -> config.useShadowDom <- false)
+    let model, dispatch = Hook.useElmish (init, update)
+    view model dispatch
+
+[<Emit("String")>]
+let StringCtor: obj = jsNative
+
+registerElement "user-profile" JsInterop.jsConstructor<ClassComponents.UserProfile> {
```

Comment:
Not sure how it works with CE but I would provide an inline generic version so users don't need to call `jsConstructor` like:

```fsharp
// This is equivalent to the current registerElement, cannot be private because the new registerElement is inlined 
// but shouldn't be used directly
let registerElementFromConstructor (name: string) (cons: obj) = ...

let inline registerElement<'T> (name: string) = registerElementFromConstructor name jsConstructor<'T>
```

Reply:
that feels better, yes we can do that